### PR TITLE
wip: Append SDEX ticker.

### PR DESCRIPTION
**Pull request by wongy91**
- What is the purpose of this Pull Request?
- What is the related issue for this Pull Request?
- [x] I have added Specs
- [x] (If implementing Market Ticker) I have verified that the `volume` refers to BASE
- [x] (If implementing Market Ticker) I have verified that the `base` and `target` is assigned correctly

### CODE REVIEW

**Comment by tmlee** [comment]
Do they really not have an API to list pairs? @wongy91

**Comment by wongy91** [comment]
ya man, the API doesn't even have a proper doc. https://github.com/irisli/stellarterm/tree/master/api
@tmlee

File: lib/cryptoexchange/exchanges/stellar_dex/stellar_dex.yml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+:pairs:
+  - :base: MOBI
```
Comment (tmlee) [inline lib/cryptoexchange/exchanges/stellar_dex/stellar_dex.yml]: Potentially explore parsing the ticker.json to create the list automatically

**Comment by wongy91** [comment]
no longer relevant.

**Pull request closed without merging**